src/player_display.cpp

```diff
@@ -300,18 +340,15 @@ static void draw_proficiencies_tab( const catacurses::window &win, const unsigne
 {
     werase( win );
     const std::vector<display_proficiency> profs = guy.display_proficiencies();
-    bool focused = curtab == player_display_tab::proficiencies;
+    const bool focused = curtab == player_display_tab::proficiencies;
     const nc_color title_color = focused ? h_light_gray : c_light_gray;
     center_print( win, 0, title_color, _( title_PROFICIENCIES ) );
-    const int height = getmaxy( win ) - 1;
-    const int width = getmaxx( win ) - 1;
-    bool draw_scrollbar = profs.size() > static_cast<size_t>( height );
-    int y = 1;
-    int start = draw_scrollbar ? line : 0;
-    for( size_t i = start; i < profs.size(); ++i ) {
-        if( y > height ) {
-            break;
-        }
+
+    const size_t height = getmaxy( win ) - 1;
```

Comment:
Using `size_t` invites all sorts of nasty surprises. I recommend always using `int` unless it cannot be done without `size_t`.